docs/samples/Microsoft.ML.Samples/Dynamic/FeatureContributionCalculationTransform.cs

```diff
@@ -85,7 +85,7 @@ public static void FeatureContributionCalculationTransform_Regression()
                 var value = row.Features[featureOfInterest];
                 var contribution = row.FeatureContributions[featureOfInterest];
                 var percentContribution = 100 * contribution / row.Score;
-                var name = data.Schema.GetColumnName(featureOfInterest + 1);
+                var name = data.Schema[(int) (featureOfInterest + 1)].Name;
```

Comment:
>(int) [](start = 39, length = 5)

why cast to int if it's already int?
To catch overflow?

Reply:
I have no idea, this is Resharper doing the work for me. I believe you are right, it's for the overflow.

---
In reply to: [242359826](https://github.com/dotnet/machinelearning/pull/1894#discussion_r242359826) [](ancestors = 242359826)